server/methods/catalog.js

```diff
@@ -793,6 +800,7 @@ Meteor.methods({
           title: "",
           type: "variant" // needed for multi-schema
         });
+        Hooks.Events.run("onProductCreate", result);
```

Comment:
All of these product hooks should have some consistency. `result` here is just going to be the results of the db call, not any information about the call.